src/platforms/common/configuration/options.mdx

```diff
@@ -281,7 +281,31 @@ The number of context lines for each frame when loading a file.
 
 <ConfigKey name="initial-scope" supported={["javascript","node"]}>
 
-Data to be set to the initial scope.
+Data to be set to the initial scope. Initial scope can be defined either as an object as follows
+
+```javascript
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  debug: true,
+  initialScope: {
+    scope.setTag("my-tag", "my value");
+    scope.setUser({id: 42, email: "john.doe@example.com"});
+  }
```

Comment:
This is not a valid code sample, you can't call `scope.setTag` it should be an object.
```suggestion
  initialScope: {
    tags: {"my-tag": "my value"},
    user: { id: 42, email: "john.doe@example.com"},
  }
```

Reply:
Oops xd